src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -71,10 +72,6 @@
   Snowpark pandas deviates from this behavior and will maintain the same behavior as pandas from versions 1.5.x.
 - Changed the import path of Snowpark pandas package to use Modin 0.28.1 instead. The new recommended import statement is `import modin.pandas as pd; import snowflake.snowpark.modin.plugin`.
 
-### New Features
-- Added partial support for `SeriesGroupBy.apply` (where the `SeriesGrouBy` is obtained through `DataFrameGroupBy.__getitem__`).
```

Comment:
Should we add back the line about `SeriesGroupBy.apply`?

Reply:
I believe that this diff is because of a merge.